openrpc.yaml

```diff
@@ -521,6 +521,263 @@ methods:
+              `true` if the wallet executed the calls atomically. `false` if the
+              wallet executed the calls non-atomically.
+            type: boolean
+          receipts:
+            description: >-
+              An array of transaction receipts. If the wallet executed the calls
+              atomically, a single receipt or an array of receipts may be returned,
+              corresponding to how the batch of calls were included onchain.
+            type: array
+            items:
+              title: Receipt
+              description: A transaction receipt object.
+              type: object
+              properties:
+                logs:
+                  description: An array of log objects.
+                  type: array
+                  items:
+                    title: Log
+                    description: An object containing information about the log.
+                    type: object
+                    properties:
+                      address:
+                        description: The address that generated the log.
+                        type: string
+                        pattern: '^0x[0-9a-fA-F]{40}$'
+                      data:
+                        description: The data of the log.
+                        type: string
+                        pattern: '^0x[0-9a-f]*$'
+                      topics:
+                        description: An array of log topics.
+                        type: array
+                        items:
+                          title: Topic
+                          description: A log topic.
+                          type: string
+                          pattern: '^0x[0-9a-f]*$'
+                status:
+                  description: >-
+                    The status of the transaction. `0x1` indicates success, and `0x0`
+                    indicates failure.
+                  type: string
+                  pattern: '^0x[0-1]$'
+                blockHash:
+                  description: The hash of the block containing the transaction.
+                  type: string
+                  pattern: '^0x[0-9a-f]{64}$'
+                blockNumber:
+                  description: The number of the block containing the transaction.
+                  type: string
+                  pattern: '^0x([1-9a-f]+[0-9a-f]*|0)$'
+                gasUsed:
+                  description: The amount of gas used by the transaction.
+                  type: string
+                  pattern: '^0x([1-9a-f]+[0-9a-f]*|0)$'
+                transactionHash:
+                  description: The hash of the transaction.
+                  type: string
+                  pattern: '^0x[0-9a-f]{64}$'
+              additionalProperties: true
+          capabilities:
+            description: Capabilities associated with the batch of calls.
+            type: object
+            additionalProperties: true
+    errors:
+      - $ref: '#/components/errors/InvalidParams'
```

Comment:
The Unauthorised error should also be here - `'#/components/errors/Unauthorized'`